docs/tasks/access-kubernetes-api/configure-aggregation-layer.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Configure the aggregation layer
+assignees:
+- lavalamp
+- cheftako
+- chenopis
+---
+
+{% capture overview %}
+
+Configuring the aggregation layer allows the Kubernetes apiserver to be extended with additional APIs, which are not part of the core Kubernetes APIs. 
+
+{% endcapture %}
+
+{% capture prerequisites %}
+
+{% include task-tutorial-prereqs.md %}
+
+**Note:** There are a few setup requirements for getting the aggregation layer working in your environment to support mutual TLS auth between the proxy and extension apiservers. Kubernetes and the kube-apiserver have multiple CAs, so make sure that the proxy is signed by the aggregation layer CA and not by something else, like the master CA.
+
+{% endcapture %}
+
+{% capture steps %}
+
+## Enable apiserver flags
+
+Enable the aggregation layer via the following apiserver flags. They may have already been taken care of by your provider.
```

Comment:
@cheftako @lavalamp Can you confirm that "apiserver" means `kube-apiserver` here?